### ISSUE TITLE
some problem about multi GPU

### ISSUE THREAD

**Issue by hyaihjq** [simonmeister/UnFlow#7]
now,i was  training "C_SYNTHIA",I found that it took the same amount of time to train 5000 iterations of two GPUs or eight GPUs. now  num_iters = 500000.The total number of iterations for two GPUs and eight GPUs is 500000? I have setted  batch_size of one GPU from 4 to 8, batch_size of two GPU is 16, batch_size of eight  GPU is 64.

**Comment by simonmeister** [comment]
Yes, if num_iters = 500000, then each GPU will do this number of iterations. If you increase the batch_size proportional to the number of GPUs, it will take the same time (or more). Increasing the batch_size can only reduce the total training time if you also use a higher learning rate and can thus train fewer iterations.

If you just want to speed up the standard schedule with two or four GPUs (what we did) without changing the learning rates and reducing iterations, you should leave the batch_size unchanged to actually reduce the amount of work to be done at each iteration per GPU.
